specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1000,91 +1000,48 @@ _https://jakarta.ee/specifications/xml-web-services/_ .
-portability to systems that do not use a POA to implement RMI-IIOP,
-applications should not depend on the fact that the _Tie_ extends the
-_Servant_ class. A Jakarta EE application that defines or uses RMI-IIOP
-objects other than enterprise beans must include such portable _Stub_
-and _Tie_ classes in the application package. _Stub_ and _Tie_ objects
-for enterprise beans, however, must not be included with the
-application: they will be generated, if needed, by the Jakarta EE product
-at deployment time or at run time.
-
-RMI-IIOP is defined by chapters 5, 6, 13, 15,
-and section 10.6.2 of the CORBA 2.3.1 specification, available at
-_http://www.omg.org/cgi-bin/doc?formal/99-10-07_ , and by the Java™
-Language To IDL Mapping Specification, available at
-_http://www.omg.org/cgi-bin/doc?ptc/2000-01-06_ .
+===== RMI-IIOP
+
+The RMI-IIOP subsystem is composed of APIs that allow for the
+use of RMI-style programming that is independent of the underlying
+protocol.  Implementations these APIs may support the Java SE native RMI
+protocol (JRMP), the CORBA IIOP protocol or any custom protocol that is
+compatible with the RMI programming restrictions (see the RMI-IIOP spec
+for details).
+
+NOTE: The requirements in this section only apply to Jakarta EE products that
+include an Enterprise Beans container with support for remote interfaces.
+
+Jakarta EE applications are required to use the RMI-IIOP APIs (specifically the
+narrow method of javax.rmi.PortableRemoteObject) when accessing
+remote Enterprise JavaBeans components, as described in the Jakarta Enterprise
+Beans 4.0 specification.  This allows Enterprise Beans and their clients to be
+protocol independent and portable to Jakarta EE implementations that may use
+CORBA or RMI as their underlying protocol.  However, there is currently no
+requirement that Enterprise Beans be accessible using the IIOP protocol itself
+or that Jakarta EE implementations support any form of distributed interoperability
+with other Jakarta EE implementations.
+
+The use of the term RMI-IIOP has existed since J2EE 1.2 and EJB 1.1, predates
+any requirements to support CORBA itself and was used to detail application
+programming restrictions necessary to allow application portability.  Requirements
+for implementations to use CORBA IIOP itself were added to J2EE 1.3 and EJB 2.0 and
+expanded the term RMI-IIOP beyond application portability and into distributed
+interoperability. These distributed interoperability requirements
+were marked Proposed Optional in Java EE 1.8 .  As of Jakarta EE 9 and Enterprise
+Beans 4.0 distributed interoperability requirements are fully removed and the term
+RMI-IIOP reverts back to a term that guarantees only application portability, not
+distributed interoperability.
+
+Jakarta EE implementations may use CORBA IIOP as their underlying protocol, however,
+such support is implementation-specific and no longer a guarantee of the Jakarta
+EE platform.
 
```

Comment:
Other sections need attention relating to Interop... Will they be separate PRs?  Or, do you plan to enhance this PR?  

Section 2.8 Interoperability (text and diagram)
Section 2.13.3 Network Protocols
Section 6.1.4 Removed Jakarta Technologies (needs clarification on CORBA, Java IDL, RMI-IIOP)
Section 6.3 Enterprise Beans 4.0 Requirements
Section 7.2 Interoperability Protocols
Section 7.2.3 Java Technology Protocols
Section 9.6 Optional Features for Jakarta EE Profiles